sisl/viz/plotly/plots/pdos.py

```diff
@@ -177,6 +183,13 @@ class PdosPlot(Plot):
                         "isSearchable": True,
                         "options": [{"value": option, "label": option} for option in ("species", "atoms", "Z", "orbitals", "spin", "n", "l", "m", "zeta")]
                     }
+                ),
+
+                FloatInput(
+                    key="scale", name="Scale",
+                    default=1,
+                    params={"min": None},
+                    help="The final DOS will be multiplied by this number. Note that it can also be a negative number."
```

Comment:
Just remove `Note that it can also be a negative number` it is a scale factor, so no need for extended documentation. :)